src/main.cpp

```diff
@@ -634,6 +654,14 @@ main(int argc, char *argv[]) {
   std::thread httpThread { nvhttp::start };
   std::thread configThread { confighttp::start };
 
+#ifdef _WIN32
+  // If we're using the default port and GameStream is enabled, warn the user
+  if (config::sunshine.port == 47989 && is_gamestream_enabled()) {
+    BOOST_LOG(fatal) << "GameStream is still enabled in GeForce Experience! This *will* cause streaming problems with Sunshine!"sv;
+    BOOST_LOG(fatal) << "Disable GameStream on the SHIELD tab in GeForce Experience or change the Port setting on the Advanced tab in the Sunshine Web UI."sv;
```

Comment:
`fatal` will terminate Sunshine, right? I'm okay with that since Sunshine wouldn't work at all anyway (except the web ui), but just wondering.

If `fatal` terminates, won't the first `fatal` prevent the second one?

Reply:
`fatal` doesn't terminate. It's just another log level.

I made it `fatal` to attract attention, but maybe @TheElixZammuto 's change will make this unnecessary.

Reply:
Actually the changes I made only shows FATAL level logs in the index.html of the Web UI, so it can be generalized for many error cases (e.g. ViGemBus not installed, Encoder Not Found, etc...)